docs/agent_hocon_reference.md

```diff
@@ -45,22 +44,19 @@ Sub-keys to those dictionaries will be described in the next-level down heading
   - [command](#command)
   - [tools (agents)](#tools-agents)
     - [External Agents](#external-agents)
-  - [llm_config](#llm_config-1)
+  - [Per agent llm_config](#per-agent-llm_config)
   - [class](#class)
   - [toolbox](#toolbox)
   - [args](#args)
   - [allow](#allow)
     - [connectivity](#connectivity)
     - [to_downstream](#to_downstream)
-      - [sly_data](#sly_data)
```

Comment:
Removed redundant (ambiguous) headers

Reply:
I'm not sure the answer is to just remove these.
The links were generated by a package and when this is rendered by GitHub (at least) they link to what is expected.